cypress/integration/smokeTests/devOnly/basic-enduser.ts

```diff
@@ -1,88 +0,0 @@
-import { goToTab } from '../../../support/commands';
```

Comment:
probably missed something simple, shouldn't this be moving to a qaOnly test, or was this already done previously?

Reply:
> probably missed something simple, shouldn't this be moving to a qaOnly test, or was this already done previously?

No, it was moved/merged to shared. It was in `devOnly` because 1.13 hadn't been deployed to staging/prod, and the Galaxy tests had a dependency on 1.13, so there was a temporary divergence. Now that 1.13 is deployed, the shared  tests can run against all environments.